Documentation/ColumnsConfig/Type/Inline/Examples.rst

```diff
@@ -136,3 +136,97 @@ added to the parent record.
 
 .. include:: /CodeSnippets/InlineUsecombinationcInline1.rst.txt
 
+.. _inline-example-field-information:
+
+Add a custom fieldInformation
+=============================
+
+We show a very minimal example which adds a custom fieldInformation for the
+inline type in tt_content. Adding a fieldWizard is done in a similar way.
+
+As explained in the :ref:`description <columns-inline>`, :code:`fieldInformation`
+or :code:`fieldWizard` must be configured within the :code:`ctrl` **for the field
+type inline** - as it is a container.
+
+.. rst-class:: bignums-xxl
+
+#. Create a custom fieldInformation
+
+   .. code-block:: php
+      :caption: EXT:myext/Classes/FormEngine/FieldInformation/DemoFieldInformation
+
+         <?php
+         declare(strict_types=1);
+         namespace Myvendor\Myexample\FormEngine\FieldInformation;
+
+         use TYPO3\CMS\Backend\Form\AbstractNode;
+
+         class DemoFieldInformation extends AbstractNode
+         {
+             public function render()
+             {
+                 $result = $this->initializeResultArray();
+
+                  // Add fieldInformation only for this field name
+                  //   this may be changed accoringly
+                  if ($fieldName === 'my_new_field') {
+                      $result['html'] = $GLOBALS['LANG']->sL(
```

Comment:
Long lines will cause horizontal scrolling in the code preview - stick to 80 max

Reply:
Squashed the commits and pushed again